vitest.config.ts

```diff
@@ -24,6 +24,7 @@ export default defineConfig({
         'src/lib/svelte-mocks/**/*',
         'src/lib/utilities/get-environment.ts',
         '**/*.test.ts',
+        '**/.svelte-kit',
```

Comment:
I think this needs to go in the `exclude` directly under `test`, not `test.coverage`